src/NuGet.Core/NuGet.Packaging/Signing/Timestamp/Timestamp.cs

```diff
@@ -53,7 +53,7 @@ public Timestamp()
         {
             UpperLimit = DateTimeOffset.Now;
             GeneralizedTime = DateTimeOffset.Now;
-            UpperLimit = DateTimeOffset.Now;
+            LowerLimit = DateTimeOffset.Now;
```

Comment:
You should capture `DateTimeOffset.Now` first and then reuse it.  For example:

```
UpperLimit = DateTimeOffset.Now;
GeneralizedTime = UpperLimit;
LowerLimit = UpperLimit;
```
This will ensure these values are always initialized to the exact same value.